app/Http/Controllers/JobController.php

```diff
@@ -67,12 +67,29 @@ public function managerIndex()
             ->withCount('submitted_applications')
             ->get();
 
+
+        /*
+            Check if job has a title. Append both bool (and french title if locale is 'en').
+            Because global fallback-locale set to true (config/translatable.php) any french translatables are set to the english copy. 'getTranslationsArray()' will retreive real value of translations.
+        */
+        $locale = app()->getLocale();
+        foreach ($jobs as &$job) {
+            $jobTranslations = $job->getTranslationsArray();
+            $hasTitle = !!$jobTranslations[$locale]['title'];
```

Comment:
Why the double '!'?

Reply:
It's just a quick way of turning it into a boolean. This is all changed though going to push up soon.

Reply:
So it was basically type casting to boolean? I'd have done a more explicit `(bool) $your_var` or used `array_key_exists()`.

Reply:
I was using it just to test something, but yes I definitely should use something more explicit.